pkg/api/mlflow/controller/metric.go

```diff
@@ -5,14 +5,15 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/G-Research/fasttrackml/pkg/database"
 	"github.com/apache/arrow/go/v11/arrow"
 	"github.com/apache/arrow/go/v11/arrow/array"
 	"github.com/apache/arrow/go/v11/arrow/ipc"
 	"github.com/apache/arrow/go/v11/arrow/memory"
 	"github.com/gofiber/fiber/v2"
 	log "github.com/sirupsen/logrus"
 
+	"github.com/G-Research/fasttrackml/pkg/database"
```

Comment:
no needed empty line.

Reply:
well spotted!

Reply:
addressed in https://github.com/G-Research/fasttrackml/pull/155/commits/95f95fda117dacbcd1f958d1e42d2f1f6749ed87